src/app/components/chat/chat-detail/chat-detail.component.ts

```diff
@@ -0,0 +1,111 @@
+import { Component, OnInit } from '@angular/core';
+import { HeaderService } from '@app/services/header.service';
+import { ActivatedRoute } from '@angular/router';
+import { Subject, BehaviorSubject, combineLatest } from 'rxjs';
+import { takeUntil, map } from 'rxjs/operators';
+import { ChatService } from '@services/chat.service';
+import { UserService } from '@app/services/user.service';
+import { UserI } from '@models/user';
+
+@Component({
+  selector: 'gtm-chat-detail',
+  templateUrl: './chat-detail.component.html',
+  styleUrls: ['./chat-detail.component.scss'],
+})
+export class ChatDetailComponent implements OnInit {
+  private unsubscribe$: Subject<void> = new Subject();
+  chatUsers$ = new BehaviorSubject<UserI[]>([]);
+  chatGroup$;
```

Comment:
It turned out this is not used, so it should be removed. Any un-used code contributes to accumulation of "junk code" that gradually pollutes the code base, so it's important to always check when submitting a PR that nothing pointless has been added. In this case, I added this line to demonstrate/test some stuff and since it wasn't contributing to the app we should now remove it.
```suggestion
```